lxd/apparmor/instance_forkproxy.go

```diff
@@ -66,11 +69,13 @@ profile "{{ .name }}" flags=(attach_disconnected,mediate_deleted) {
   deny @{PROC}/@{pid}/cgroup r,
   deny /sys/module/apparmor/parameters/enabled r,
   deny /sys/kernel/mm/transparent_hugepage/hpage_pmd_size r,
+  deny /sys/devices/virtual/dmi/id/product_uuid r,
 
 {{- if .snap }}
   # The binary itself (for nesting)
   /var/snap/lxd/common/lxd.debug      mr,
   /snap/lxd/*/bin/lxd                 mr,
+  {{ .rootPath }}/run/systemd/resolve/stub-resolv.conf r,
```

Comment:
Does that really need to be inside that `{{- if .snap }}`? I would assume it can go in the section where you added access to `/etc/machine-id`.

Reply:
Done